utilities/backupable/backupable_db_test.cc

```diff
@@ -894,14 +894,17 @@ class BackupableDBTest : public testing::Test {
 
   void AssertDirectoryFilesMatchRegex(const std::string& dir,
                                       const std::regex& pattern,
+                                      const std::string file_type,
                                       int minimum_count) {
     std::vector<FileAttributes> children;
     ASSERT_OK(file_manager_->GetChildrenFileAttributes(dir, &children));
     int found_count = 0;
     for (const auto& child : children) {
-      const std::string match("match");
-      ASSERT_EQ(match, std::regex_replace(child.name, pattern, match));
-      ++found_count;
+      if (child.name.find(file_type) != std::string::npos) {
+        const std::string match("match");
+        ASSERT_EQ(match, std::regex_replace(child.name, pattern, match));
```

Comment:
The reason it is written this way is so that you get a decent message on failure, rather than expected "true" found "false."

Reply:
Perhaps we should change to

    const std::string match("Great success!");

😆  (Obviously it would be nice to have ASSERT_MATCHES_REGEX(str, pattern) but I didn't implement that. Related: I also wish ASSERT_OK gave a better failure message.)

Reply:
Got it. Actually, `googletest` supports custom failure messages via the stream operator; in this specific case, we could do

```
ASSERT_TRUE(std::regex_match(child.name, pattern)) << "File name " << child.name << " does not match regex " << pattern;
```

But `Great success` also works. :)